pom.xml

```diff
@@ -150,7 +150,7 @@
         <org.eclipse.persistence.eclipselink.version>2.7.7</org.eclipse.persistence.eclipselink.version>
         <org.eclipse.persistence.extension.version>2.7.7</org.eclipse.persistence.extension.version>
         <org.eclipse.persistence.javax.persistence.version>2.2.1</org.eclipse.persistence.javax.persistence.version>
-        <org.flywaydb.version>5.2.4</org.flywaydb.version><!--Latest 6.4.0 needs changes-->
```

Comment:
The fact there is a comment here stated that 6.4.0 needs changed is concerning when we are going to 9.3.1

Reply:
I've removed the comment and flyway version changed to '8.5.13'.